codecov.yml

```diff
@@ -10,3 +10,6 @@ coverage:
         target: 100%
 
 comment: false
+
+ignore:
+ - "celery_utils/management/commands/drop_djcelery_tables.py"
```

Comment:
I feel confident ignoring this for coverage reporting, as it'd be super not worth it to set up a unit test to cover the weird one-time version upgrade scenario this command is designed for.

I have manually created the intended environment several times locally, and it works as expected in either empty or non-empty cases.